ruby_on_rails/environment_protection.md

```diff
@@ -0,0 +1,26 @@
+# Staging Environment Protection
+
+To protect Renuo from a lawsuit for using copyrighted material, staging applications should be protected using HTTP Basic auth to prevent unwanted traffic.
```

Comment:
I'd not write this so explicitly here. I'd say that the main reason is that by doing so the develop and testing environments cannot be accessed publicly but only from us and our customers